src/main/kotlin/com/wgtwo/example/voicemail/VoicemailMain.kt

```diff
@@ -8,15 +8,21 @@ import com.github.ajalt.clikt.parameters.options.required
 
 fun main(args: Array<String>) = Voicemail.main(args)
 
-object Voicemail: CliktCommand() {
+object Voicemail : CliktCommand() {
     override fun run() = Unit
+
     init {
         subcommands(List, Play, MarkRead, Delete)
     }
 }
 
-object List: CliktCommand() {
-    val msisdn by option("-m", "--msisdn", help = "Msisdn", envvar = "MSISDN").required()
+object List : CliktCommand() {
+    val msisdn by option(
```

Comment:
doesn't msisdn mean that the plus is omitted, which is a contradiction to e164? Rename the val to e162Number (and drop the comment)?

Reply:
Done!

Reply:
Rename this val too? And the `-m` -> `-e`?